api/config/service/aws/v1/aws.proto

```diff
@@ -9,9 +9,26 @@ import "validate/validate.proto";
 message Config {
   repeated string regions = 1 [ (validate.rules).repeated = {min_items : 1} ];
   ClientConfig client_config = 2;
+  ClientSettings client_settings = 3;
 }
 
 message ClientConfig {
   // If not set explicity, retries default to 0
   int32 retries = 1 [ (validate.rules).int32.gte = 0 ];
 }
+
+message ClientSettings {
```

Comment:
Should we just put these fields into the `ClientConfig` if so why not?

Reply:
I originally separated them to keep AWS client config variables like `HTTPClient`, `Region`, `Retryer`, `SocketTimeout`, etc distinct from the custom internal settings/constants used by the workflows. Bad naming on my part! Should I just combine all fields under `ClientConfig` or rename ClientSettings?

Reply:
Yeah good question since there are a few configuration options for dynamodb maybe we can put it under its own top level message maybe something like this? What do you think?

```proto
message Config {
  repeated string regions = 1 [ (validate.rules).repeated = {min_items : 1} ];
  ClientConfig client_config = 2;
  DynamodbConfig dynamodb_config = 3;
}

message DynamodbConfig {
...
}
```

Reply:
Looks good to me! Sets up the config struct nicely for additional datastores configs if needed in the future too